### ISSUE TITLE
Loss looks strange on training

### ISSUE THREAD

**Issue by WaterKnight1998** [NathanUA/U-2-Net#26]
At training validation loss is bouncing which looks very strange.
Here I attach you a plot with training and validation loss curves:
![image](https://user-images.githubusercontent.com/41203448/82489772-77b72880-9ae2-11ea-9fe2-b37517ba90c4.png)

**Comment by WaterKnight1998** [comment]
The learning rate was very big!

**Comment by bluesky314** [comment]
@WaterKnight1998 Big as in?

**Comment by WaterKnight1998** [comment]
> @WaterKnight1998 Big as in?

Don´t understand

**Comment by shoutOutYangJie** [comment]
> > @WaterKnight1998 Big as in?
> 
> Don´t understand

where can download your dataset.  I don't have 'im_aug' and 'gt_aug'.
